api/all/src/main/java/io/opentelemetry/api/trace/TraceFlags.java

```diff
@@ -8,47 +8,128 @@
 import javax.annotation.concurrent.Immutable;
 
 /**
- * Helper methods for dealing with trace flags options. These options are propagated to all child
- * {@link Span spans}. These determine features such as whether a {@code Span} should be traced. It
- * is implemented as a bitmask.
+ * Helper methods for dealing with trace flags options. A valid trace flags is a 2 character
+ * lowercase hex (base16) String.
+ *
+ * <p>These options are propagated to all child {@link Span spans}. These determine features such as
+ * whether a {@code Span} should be traced. It is implemented as a bitmask.
```

Comment:
```suggestion
 * whether a {@code Span} should be traced.
```

Don't think we need implementation detail here (it wasn't so clear to me either way)

Reply:
Done.